.github/styles/Vocab/Docs/accept.txt

```diff
@@ -11,6 +11,8 @@ Avro
 Conduktor
 [Cc]onnect
 Couchbase
+CPU
```

Comment:
Are you sure it is needed? We use CPU in other docs, seems to be a known word already. Could you double check that you get errors if you remove it?

Reply:
Thanks! turns out if I use all caps, CPU, SMTP, CRON all passed!